src/main/java/gregtech/api/modules/GregTechModule.java

```diff
@@ -0,0 +1,25 @@
+package gregtech.api.modules;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface GregTechModule {
```

Comment:
Where is this even used?

Reply:
Its used for module registration, done via ASM discovery (and then used periodically for Module metadata)

Reply:
See ModuleManager#configureModules